lib/services/services.c

```diff
@@ -146,6 +146,7 @@ expand_resource_class(const char *rsc, const char *standard, const char *agent)
     return expanded_class;
 }
 
+#if SUPPORT_NAGIOS
```

Comment:
This was actually uncovered by recent CI changes. Previously we were pretty much using only the configure script defaults. I made it a point to test several of the key options on different platforms to give us more coverage, including --with-nagios=no.

I'm pretty sure I manually tried building with --with-nagios=no, so I'm not sure why I didn't catch this already, but in any case I'm glad we got it :)